hal.c

```diff
@@ -94,6 +96,10 @@ void hal_init(uint32_t mclkFreq)
     //CC1352R UART
     GPIO_setAsPeripheralModuleFunctionInputPin(GPIO_PORT_P4, GPIO_PIN4 | GPIO_PIN5);
 
+    //I2C
```

Comment:
Why is this commented out? Shouldn't we have the proper pin configuration here?

Reply:
It has the same function as the pins in hal_ext_i2c. Of course, we can keep it, and it won't raise an exception.
```
GPIO_setAsPeripheralModuleFunctionInputPin(GPIO_PORT_P4, GPIO_PIN1 | GPIO_PIN2);
```